visual-recognition/src/main/java/com/ibm/watson/visual_recognition/v4/model/TrainingEvent.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (C) Copyright IBM Corp. 2019.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.visual_recognition.v4.model;
+
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.cloud.sdk.core.service.model.GenericModel;
+
+/**
+ * Details about the training event.
+ */
+public class TrainingEvent extends GenericModel {
+
+  /**
+   * Trained object type. Only `objects` is currently supported.
+   */
+  public interface Type {
+    /** objects. */
+    String OBJECTS = "objects";
+  }
+
+  /**
+   * Training status of the training event.
+   */
+  public interface Status {
+    /** failed. */
+    String FAILED = "failed";
+    /** succeeded. */
+    String SUCCEEDED = "succeeded";
+  }
+
+  private String type;
+  @SerializedName("collection_id")
+  private String collectionId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)